L1Trigger/L1TGlobal/src/CorrCondition.cc

```diff
@@ -321,6 +321,14 @@ const bool l1t::CorrCondition::evaluateCondition(const int bxEval) const {
   int etBin1 = 0;
   double et1Phy = 0.;
 
+  // Added by R.Cavanaugh for displaced muons
```

Comment:
Please remove meaningless comments for understanding the code.  Git versioning system keeps track of authors of commits.

Reply:
@rekovic 
Many thanks for the helpful request to remove comments.  Done.